docs/tutorials/hackathon-prep-course/7-obtaining-cycles.md

```diff
@@ -162,7 +152,7 @@ If you manually converted ICP tokens into cycles, and your cycles wallet doesn't
 dfx ledger top-up 5o6tz-saaaa-aaaaa-qaacq-cai --network ic --amount 1.005
```

Comment:
```suggestion
dfx ledger top-up <CYCLE_WALLET_CANISTER_ID> --network ic --amount 1.005
```

Reply:
we should add a previous step explaining how to get the canister id
```
dfx identity --ic get-wallet
```

Reply:
where is the `vite-motoko-react` project coming from that we are deploying here? i think its helpful to link to it again, even if its used in previous sections of the guide